index.html

```diff
@@ -3,47 +3,294 @@
+        // This anonymous function is responsible for comparing two values a and b. If it returns a positive
+        // number, we're saying that a should come after b in the array. A negative number means that b
+        // should come after a. A zero means that the two are equal. This function is using the 'enrollment'
+        // property of each observation of data for this comparison. You'll need to change 'enrollment' to
+        // whatever property you want to sort your data on.
+        //
+        // Also, note that we're sorting the data in place, changing, or mutating, the 'data' variable. See
+        // the note in the README about how this compares to the filter() and slice() operations that we'll
+        // use for the next two data transforms.
+      } 
+      else if (options.sorted === 'desc') {
+        data.sort(function (a, b) { return  b.Victims- a.Victims; });
+        // Note that this is just the opposite of the above.
+      }
+        }
+    // Filter victims by region
+    // Filter: Step1 : Preparing data
+  
+    var data = conflict[0];
+    //putting region in a separate array 
+    // Filter: Step2: Creating the function
+    data = conflict.filter(function (a) { return d.Region > 0; });
+    // Filter: Step3: Creating the filters - How the function is going to work 
+   if (options.filtered === 'Region 1') {
+        data = conflict.filter(function (d) { return d.Region === "1" })
+        } else if (options.filtered === 'Region 2') {
+      data = conflict.filter(function (d) { 
+        return d.Region === "2"})
+    } else if (options.filtered === 'Region 3') {
+      data = conflict.filter(function (d) { 
+        return d.Region === "3"})
+    } else if (options.filtered === 'Region 4') {
+      data = conflict.filter(function (d) { 
+        return d.Region === "4"})
+    } else if (options.filtered === 'Region 5') {
+      data = conflict.filter(function (d) { 
+        return d.Region === "5"}
+  }
+  
+
+    // Select only the Departments from the current page (slice)
+    var table = document.getElementById('data-table');
+    var tbody = table.getElementsByTagName('tbody')[0];
+    // Because getElementsByTagName returns an array of DOM nodes (technically, it's an HTMLCollection),
+    // we need to select the first item in the array with the [0].
+    // Remove any existing rows
+    tbody.innerHTML = '';
+    // Add a new row for each datum (item in the 'data' array)
+    data.forEach(function (d) {
+      var row = document.createElement('tr');
+      // This is like create a new <tr></tr> in HTML, but it doesn't yet exist in the DOM, so we can't
+      // see it. At the bottom of this forEach() loop, we append it to an element that is in the DOM,
+      // at which point it appears. We'll follow this same pattern for each of the <td></td> elements
+      // we create below.
+      
+
+      var Department = document.createElement('td');
```

Comment:
We want to avoid using variable names that start with capital letters unless they are all-caps CONSTANTS, unchanging values like `MAX_VICTIMS`, or a [constructor function](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/new). Instead, the standard style is to use camelCase, with the first letter of each word after the first capitalized.

Side note, it's also useful for your data columns to be named in this way, although snake_case is also acceptable. It's still better if they don't start with a capital letter, though.